source/_components/light.mipow.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Mipow Color"
+description: "Instructions how to setup Mipow color BT le bulb within Home Assistant."
+date: 2015-10-25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mipow.png
```

Comment:
*No applicable to this line but can't comment on the picture.* I would suggest chaining the logo to just the company logo if possible.